### PR TITLE
Use ::class instead of FQCN

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | -
| Tickets       | -
| License       | MIT
| Doc PR        | -

I didn't do it in tests because there are a ton of occurrences, so it would be very long to review. Also, I didn't do it in the VarDumper component because it seems intentional.

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function __construct(ValidatorInterface $validator, bool $legacyErrorMess
     {
         $this->legacyErrorMessages = $legacyErrorMessages;
 
-        $metadata = $validator->getMetadataFor('Symfony\Component\Form\Form');
+        $metadata = $validator->getMetadataFor(\Symfony\Component\Form\Form::class);
```
Comment (derrabus) [inline src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php:39]: I think it's fine to import this class.

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php (hunk 1 above)
Comment (fancyweb) [reply src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php:39]: `ValidatorExtension` already uses `Symfony\Component\Form\Extension\Validator\Constraints\Form;` so we would need to alias one.

Comment (derrabus) [reply src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php:39]: All right, let's keep it that way then.

**Comment by nicolas-grekas** [comment]
Thank you @fancyweb.

**Pull request merged**